dask_xgboost/core.py

```diff
@@ -303,6 +303,19 @@ def predict(client, model, data):
     return result
 
 
+def _xgb_regressor_fit(obj, X, y):
+    client = default_client()
+    xgb_options = obj.get_xgb_params()
+    obj._Booster = train(client, xgb_options, X, y,
+                          num_boost_round=obj.get_num_boosting_rounds())
+    return obj
+
+
+def _xgb_regressor_predict(obj, X):
```

Comment:
In `master`, `def _*` methods are reserved for Dask utilities, and parent calls which we don't intend to expose to the user.

Method useful for a particular class object are encapsulated by that class, and generally not hidden.